src/js/backup_restore.js

```diff
@@ -144,8 +144,12 @@ function configuration_backup(callback) {
                 configuration.LED_STRIP = jQuery.extend(true, [], FC.LED_STRIP);
                 configuration.LED_COLORS = jQuery.extend(true, [], FC.LED_COLORS);
                 configuration.BOARD_ALIGNMENT_CONFIG = jQuery.extend(true, {}, FC.BOARD_ALIGNMENT_CONFIG);
-                configuration.CRAFT_NAME = FC.CONFIG.name;
-                configuration.DISPLAY_NAME = FC.CONFIG.displayName;
+                configuration.CRAFT_NAME = FC.CONFIG.craftName;
+                if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)) {
+                    configuration.PILOT_NAME = FC.CONFIG.pilotName;
+                } else {
+                    configuration.DISPLAY_NAME = FC.CONFIG.displayName;
+                }
```

Comment:
Please keep backwards compatibility and apply to this the PR

```suggestion
                if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_45)) {
                    configuration.CRAFT_NAME = FC.CONFIG.craftName
                    configuration.PILOT_NAME = FC.CONFIG.pilotName;
                } else {
                    configuration.CRAFT_NAME = FC.CONFIG.name;
                    configuration.DISPLAY_NAME = FC.CONFIG.displayName;
                }
```